crmsh/cibconfig.py

```diff
@@ -858,9 +858,11 @@ def parse_cli_to_xml(cli, oldnode=None):
     complete = False
     comments = []
     if isinstance(cli, str):
+        utils.auto_convert_role = False
```

Comment:
I'm kind of confused about the context here and why automatic conversion of the role names would break parsing and impact `crm configure show`. But as long as this resolves that :)